src/config/koa.js

```diff
@@ -13,17 +13,12 @@ const routeErrorHandler = async (ctx, next) => {
   try {
     await next();
   } catch (error) {
-    ctx.status = error.statusCode || error.status || 500;
+    const status = error.status || 500;
+    const errors = error.errors || error.message;
 
-    if (!ctx.body) {
-      ctx.body = {
-        errors: {
-          _global: [error.message],
-        },
-      };
-    }
-
-    logger.error(ctx.body);
+    ctx.status = status;
+    ctx.body = { errors: error.errors || { _global: 'Something went wrong.' } };
```

Comment:
I think it will be good to expose all errors to client, if it's not production environment.

```
    const status = error.status || 500;
    const errors = error.errors || error.message;

    ctx.status = status;
    ctx.body = process.env.APP_ENV === 'production'
      ? { errors: error.errors || { _global: ['Something went wrong.'] } }
      : { errors: error.errors || { _global: [error.message] } };
```